vars/README.md

```diff
@@ -194,9 +195,11 @@ Builds the Docker image for Kibana, from a branch or a pull Request.
 ```
 buildKibanaDockerImage(refspec: 'main')
 buildKibanaDockerImage(refspec: 'PR/12345')
+buildKibanaDockerImage(refspec: 'cf25ac3d1f8edff8f20003add707bfdc85d89fff', depth: 10)
+
 ```
 
-* refspec: A branch (i.e. main), or a pull request identified by the "pr/" prefix and the pull request ID.
+* refspec: A branch (i.e. main), a commit SHA, a tag, or a pull request identified by the "pr/" prefix and the pull request ID.
```

Comment:
Tags are not supported, for release tags you can use the official image and for other tags you can use the commit SHA

```suggestion
* refspec: A branch (i.e. main), a commit SHA, or a pull request identified by the "pr/" prefix and the pull request ID.
```

Reply:
I will fix it in other PR is not related with this one

Reply:
This is unrelated to this PR, since it was generated, let's fix the comment in a follow up